src/utils/runtimeBlockNumber.ts

```diff
@@ -0,0 +1,8 @@
+export default function getRuntimeBlockNumber() {
+  if (process.env.NODE_ENV !== 'production') {
+    return process.env.NEXT_PUBLIC_FORK_BLOCK_NUMBER
+      ? BigInt(process.env.NEXT_PUBLIC_FORK_BLOCK_NUMBER)
+      : undefined;
+  }
+  return undefined;
```

Comment:
what about `latest` which is I guess the same, but just imho would look better

Reply:
Hmm sure I guess